README.md

```diff
@@ -89,4 +89,4 @@ Contrubition guidelines are available in the [contributing file](.github/contrib
 
 If you want to aid the project in other ways, consider supporting the project on [Patreon](https://patreon.com/marquiskurt). You can also [view all of our contributors](patreon.md) that help make Hyperspace possible.
 
-If you have Matrix, you can join the Hyperspace community ([+hyperspace-masto:matrix.org](https://matrix.to/#/+hyperspace-masto:matrix.org)).
\ No newline at end of file
+If you have Matrix, you can join the Hyperspace community ([+hyperspace-masto:matrix.org](https://matrix.to/#/+hyperspace-masto:matrix.org)).
```

Comment:
There's no need to change this line, and it appears there has been no significant changes have been made.

Reply:
This is a change that’s made automatically by lots of code editors (including mine) and it’s a pain in the ass to remove this.

Also there is a reasoning behind this convention:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

But I can remove it if you don’t like it.